build.gradle.kts

```diff
@@ -88,33 +84,30 @@ gradlePlugin {
 }
 
 java {
-    sourceCompatibility = JavaVersion.VERSION_11
-    targetCompatibility = JavaVersion.VERSION_11
+    toolchain {
+        languageVersion = JavaLanguageVersion.of(17)
+    }
 }
 
 // set correct project status
 if (project.version.toString().endsWith("-SNAPSHOT")) {
     status = "snapshot'"
 }
 
-detekt {
-    source = files("src/main/kotlin")
-    config = files("detekt.yml")
-}
-
+val buildDir = layout.buildDirectory.asFile.get()
 tasks {
     withType<Test>().configureEach {
-        testLogging tl@{
-            tl@this.showStandardStreams = true
-            tl@this.showStackTraces = true
-            tl@this.events(org.gradle.api.tasks.testing.logging.TestLogEvent.FAILED)
+        testLogging {
+            showStandardStreams = true
+            showStackTraces = true
+            events(org.gradle.api.tasks.testing.logging.TestLogEvent.FAILED)
         }
 
         this.javaLauncher.set( project.javaToolchains.launcherFor {
-            languageVersion.set(JavaLanguageVersion.of(11))
+            languageVersion.set(JavaLanguageVersion.of(17))
         })
 
-        systemProperty("intershop.gradle.versions","8.0.2")
+        systemProperty("intershop.gradle.versions","8.5")
```

Comment:
If we still aim to be compatible with older versions (which should be possible since you didn't do any breaking changes) the old version should stay in there and continue to be tested against. If we don't want to stay compatible, you should also adapt the README.